EdFi.Buzz.Installer/eng/Windows/Application/appinstalls.psm1

```diff
@@ -40,6 +40,9 @@ function Install-BuzzApp {
         Import-Module -Force $folders.modules.invoke("packaging/nuget-helper.psm1")
         $installFolder = Get-NuGetPackage @installparams
 
+        Copy-Item -Path "./AppSharedLibrary/Buzz-App-Install.psm1" -Destination (Join-Path $installFolder "Windows")
+        Copy-Item -Path "./AppSharedLibrary/nuget-helper.psm1" -Destination (Join-Path $installFolder "Windows")
+
```

Comment:
I was wondering how we could share those common scripts. I had thought about creating yet another NuGet package like EdFi.AppCommon.Installer - but felt that was too much. This is a simple, straight-forward solution.